privacy/scrubber_test.go

```diff
@@ -586,6 +586,37 @@ func TestScrubIPV6Lowest32Bits(t *testing.T) {
 	}
 }
 
+func TestAnonymizeIpv6(t *testing.T) {
+	testCases := []struct {
+		IP          string
+		cleanedIP   string
+		description string
```

Comment:
Please change `description` to `name` to match the arguments of `t.Run` and provide a simple and short test name per case. These tests do not have a `description`.

Alternatively for this test, using the IP as the test name might work well.